.gitignore

```diff
@@ -12,6 +12,7 @@ _test
 
 # MS VSCode
 .vscode
+__debug_bin
```

Comment:
Isn't it `__debug.bin`?
```suggestion
__debug.bin
```

Reply:
No. You can check [here](https://github.com/go-gitea/gitea/pull/8739/commits/20d6702bcd9490f1fc6d1240abc442338d396f20)

Reply:
Ah, okay. I misread the first time. 🙃

Reply:
:)

Reply:
@jolheiser I also committed previous pr as `__debug.bin` (as message) :laughing: